main/main.cpp

```diff
@@ -394,7 +394,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	Vector<String> breakpoints;
 	bool use_custom_res = true;
 	bool force_res = false;
-	bool found_project = false;
+	bool found_project ATTRIBUTE_UNUSED = false;
```

Comment:
If it's unused, it should be removed.

But in this case it's not unused, it's just that it's used only within `#ifdef TOOLS_ENABLED`, so you should enclose it in `#ifdef TOOLS_ENABLED` here and at line 749.